symengine/tests/cwrapper/test_cwrapper.c

```diff
@@ -83,11 +83,31 @@ void test_basic() {
     basic_free_heap(y);
 }
 
+void test_complex() {
+    basic e;
+    basic f;
+    char* s;
+    basic_init(e);
+    basic_init(f);
```

Comment:
You are right. I am sorry, I shouldn't have overlooked this. 
I couldn't use Debug mode as my master [throws errors](https://gitter.im/sympy/symengine) while building in Debug mode. I am still trying to figure this out.